apps/signing/tests/test_views.py

```diff
@@ -124,6 +126,35 @@ def test_version_already_uploaded(self):
         assert response.status_code == 409
         assert response.data['error'] == 'Version already exists.'
 
+    @mock.patch('devhub.views.auto_sign_version')
+    def test_version_added_is_experiment(self, sign_version):
+        self.grant_permission(self.user, 'Experiments:submit')
+        guid = 'experiment@xpi'
+        qs = Addon.unfiltered.filter(guid=guid)
+        assert not qs.exists()
+        response = self.put(
+            addon=guid, version='0.1',
+            filename=os.path.join('apps/files/fixtures/files/experiment.xpi'))
+        assert response.status_code == 201
+        assert qs.exists()
+        addon = qs.get()
+        assert addon.has_author(self.user)
+        assert not addon.is_listed
+        assert addon.status == amo.STATUS_LITE
```

Comment:
This is failing on Travis. It normally gets set in the [`submit_file` task](https://github.com/mozilla/olympia/blob/9a9231e0c79b99e0ac0e42e74d6b65a970c0176a/apps/devhub/tasks.py#L83) but it probably isn't because `FileUpload.passed_all_validations` isn't `True`.